optuna/samplers/_cmaes.py

```diff
@@ -144,6 +144,10 @@ def objective(trial):
                 versions without prior notice. See
                 https://github.com/optuna/optuna/releases/tag/v2.1.0.
 
+        popsize:
+            A population size of CMA-ES.  When setted ``restart_strategy = 'ipop'``, this is used
```

Comment:
minor typo: 

```suggestion
            A population size of CMA-ES. When set ``restart_strategy = 'ipop'``, this is used
```

Removing one space from the double spaces is also pointed out by @c-bata, but let me include it in this comment for the sake of completeness.